src/excmds.ts

```diff
@@ -3487,7 +3488,7 @@ import * as hinting from "@src/content/hinting"
         - -A save-as the linked image
         - -; focus an element
         - -# yank an element's anchor URL to clipboard
-        - -c [selector] hint links that match the css selector
+        - -c* [selector] hint links that match the css selector
```

Comment:
What does the star mean?

Reply:
I saw your reply by email but apparently can't see it here. I think it's obvious that the flag can be combined. I would interpret the * as being a literal part of the flag.